Dockerfile

```diff
@@ -1,7 +1,11 @@
 FROM golang:1.12-alpine3.10 as build
+ENV GO111MODULE=on
+
+
 RUN apk add --no-cache make
 WORKDIR /go/src/github.com/uswitch/kiam
 ADD . .
+RUN go mod download
```

Comment:
go build takes care of downloading the dependencies, we don't need `go mod download`

Reply:
I was referring mostly to what kubebuilder does in caching docker layers: https://github.com/kubernetes-sigs/kubebuilder/blob/master/testdata/project-v2/Dockerfile#L8

Reply:
cool, TFS